src/lib/bignumber/bignumber.ts

```diff
@@ -6,12 +6,28 @@ export type BN = BigNumber
 
 export const bn = (n: BigNumber.Value, base = 10): BN => new BigNumber(n, base)
 
-export const bnOrZero = (n: BigNumber.Value | null | undefined): BN => {
-  const value = bn(n || 0)
+export const bnOrZero = (n: BigNumber.Value | null | undefined, base = 10): BN => {
```

Comment:
a little skeptical of the necessity of this change, can you outline why this is necessary?

Reply:
Yeah, unless this is *akchually* consumed, I'd keep the helper as is.
More than not needing this change, arity changes aren't as small of a change as they might seem at first glance (what if a consumer is doing e.g `someMaybeUndefinedItems.map(bnOrZero)` and `index` is passed? This would silently break things, and we've been burned with `parseInt` second `radix` param before